site/app/templates/misc/MarkdownArea.twig

```diff
@@ -108,15 +108,4 @@
         <div id="{{ preview_div_id }}" class="fill-available markdown-preview" style="min-height:{{ min_height is defined ? min_height : "100px" }}; max-height:{{ max_height is defined ? max_height : "300px" }}" hidden></div>
     </div>
 </div>
-<script>
-    $('#{{markdown_area_id}}').ready(function() {
-        enableTabsInTextArea("#{{markdown_area_id}}");
-        const markdown_area = $('#{{markdown_area_id}}').closest('.markdown-area')
-        markdown_area.find('[data-initialize-preview="1"]').trigger('click');
-    });
-    $('.markdown-mode-tab').on('click', function(){
-        $(this).addClass('active');
-        const markdown_area = $(this).closest('.markdown-area');
-        markdown_area.find('.markdown-mode-tab').not(this).removeClass('active');
-    });
-</script>
\ No newline at end of file
+<script src="/js/markdown.js"></script>
```

Comment:
I think it would make sense to add this file to every request via `setInternalResources()` in `Output.php`.